dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf

```diff
@@ -0,0 +1,32 @@
+resource "kafka_topic" "uswitch.data.v1" {
+  name = "customer-proposition.uswitch.data.v1"
+
+  replication_factor = 3
+  partitions         = 15
```

Comment:
Do you really need 15 partitions? 
If this is a topic that's gonna be migrated then it should be the same as the one already existing.

Reply:
>If this is a topic that's gonna be migrated then it should be the same as the one already existing.

Yep, the current topic is 15 partitions https://github.com/utilitywarehouse/kubernetes-manifests/blob/d050a8ecb9a0efe3cdaf7a8f4bcbcb71d6840e65/prod-merit/customer-proposition/kafka-topic-applier/topics.yaml#L94-L94

I don't really have a good idea/understanding of what a sensible number of partitions would be for a given scenario, our team has always used `15` as a default. this topic doesn't have a whole lot of data in it so 15 is probably overkill.

Reply:
i don't suppose you have any docs on hand that would help (in the future) determining a good number of partitions for different scenarios?

Reply:
>i don't suppose you have any docs on hand that would help (in the future) determining a good number of partitions for different scenarios?

Not really but trying to explain:
In a nutshell the number of partitions determines the number of concurent consumers that you can have, which may influence the consuming throughput

Reply:
Gotcha, conceptually that makes sense. I still don't really know how to go "OK we're expecting x number of concurrent consumers at a given time so we should have Y number of partitions".

Reply:
I guess it depends on the use case